invoke-ai/InvokeAI
# fix: build frontend in  pypi-release workflow

## What type of PR is this? (check all applicable)



## Have you discussed this change with the InvokeAI team?

## Description

This was missing, resulting in the 3.5.0rc1 having no frontend.

## Related Tickets & Documents



- Discord installer thread: https://discord.com/channels/1020123559063990373/1149513695567810630/1185200427717898260
- Comments from here in the release chat: https://discord.com/channels/1020123559063990373/1020123559831539744/1185004017521279007

## QA Instructions, Screenshots, Recordings

I've run this locally and it works (I commented out the final steps of the workflow that do PyPi stuff to ensure I didn't accidentally deploy something).

You can run the workflow locally with https://github.com/nektos/act. Suggest using the `gh` CLI version, its very easy to set up if you have the github CLI installed. Then you can run `gh act -W .github/workflows/pypi-release.yml` to run the workflow locally in a docker image.

I don't know this local action runner would actually release to PyPi - as mentioned, I commented those steps out when testing - but it does successfully do both frontend and backend builds.



## Merge Plan

This needs @lstein 's approval.



## [optional] Are there any post deployment tasks we need to perform?

Cut an RC2
